### DISCUSSION TITLE
提案: 公式ページからユーザーズフォーラム／本リポジトリへのリンクの追加

### DISCUSSION THREAD

**Discussion by takasehideki** [toppers/users-forum#133] [General]
TOPPERSプロジェクトの公式ページからGitHubのこのOrganizationへのリンクが無いように思われます．見逃していたらすみません．
https://toppers.jp/index.html

特に[Discussions](https://github.com/orgs/toppers/discussions)は，従来のMLからの移行先であるユーザーズフォーラムとして，そろそろ認知・機能してきているのではないかと思われます．左側のメニューバーにアイコンを設けるレベルで追加して良いのではと思われます．ご検討ください．

**Comment by takasehideki** [comment]
さっそくすみません，見落としていただけでした．
左側メニューバーの コミュニティ情報 > ユーザーズフォーラム を辿るといけなくはないですね．ただし導線がよろしくないと思いますので，「左側のメニューバーにアイコンを設けるレベルで追加」は変わらずの提案とさせてください．

**Comment by hiroakitakada** [comment]
ご意見，ありがとうございます。担当者と相談します。
